arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -906,24 +999,80 @@
           }
         ],
         "responses": {
-          "202": {
-            "description": "Accepted the request to deactivate the device."
+          "200": {
+            "description": "The Alert settings.",
+            "schema": {
+              "$ref": "#/definitions/AlertSettings"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "DeviceSettings"
+        ],
+        "description": "Creates or updates the alert settings of the specified device.",
+        "operationId": "DeviceSettings_CreateOrUpdateAlertSettings",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "deviceName",
```

Comment:
Hmm, ok, I'm not sure what order the portal follows, my comment was about the order of the parameters in the SDK methods (why would it affect deployed APIs? or did you mean the SDK APIs?), if we do want to keep the order as is for the SDK and you're not planning to update it later, then it'd be your intentional call. In general, for other Azure Services we recommend the convention I mentioned before, following the order the parameters show up in the path.